scipy/spatial/distance.pyi

```diff
@@ -153,6 +153,10 @@ def kulsinski(
     u: ArrayLike, v: ArrayLike, w: Optional[ArrayLike] = ...
 ) -> np.float64: ...
 
+def kulczynski1(
+    u: ArrayLike, v: ArrayLike, w: Optional[ArrayLike] = ...
```

Comment:
But also bool no?

Reply:
This is following the same pattern as other functions. However, I checked and found that other functions also accept `bool` and not only `ArrayLike`. So, should we make the change at other places too? If yes, then in this PR or in a follow up?

Reply:
Actually this is ok as `ArrayLike` take care of `bool` as well.